vxmessenger/transport.py

```diff
@@ -175,6 +183,16 @@ class MessengerTransport(HttpRpcTransport):
     def setup_transport(self):
         yield super(MessengerTransport, self).setup_transport()
         self.pool = HTTPConnectionPool(self.clock, persistent=False)
+        self.BATCH_API_URL = self.config.get('outbound_url',
+                                             'https://graph.facebook.com')
```

Comment:
Derive this from the existing url e.g. `https://graph.facebook.com/v2.6/me/messages`, so as to minimise breakage when upgrading.